lib/components/Button/Button.tsx

```diff
@@ -18,19 +16,16 @@ export interface ButtonProps {
 }
 
 export const Button = ({ label, onClick, variant = 'base', disabled = false }: ButtonProps) => {
+  const className = `m-2 px-4 py-2 font-bold
+      ${variant === 'base' ? 'bg-jod-base text-jod-white' : ''}
+      ${variant === 'primary' ? 'bg-jod-primary text-jod-white' : ''}
+      ${variant === 'outlined' ? 'border border-jod-black text-jod-black' : ''}
+      ${variant === 'text' ? 'text-jod-black' : ''}
+      ${disabled ? 'cursor-not-allowed opacity-50' : ''}`
+    .replace(/\s+/g, ' ')
```

Comment:
Not so familiar with regexp, but does it remove whitespaces between the values? As `trim()` only removes from start and end, IIRC.